src/main/docker/monitoring.yml

```
@@ -1,23 +1,26 @@
 # This configuration is intended for development purpose, it's **your** responsibility to harden it for production
-version: '3.8'
+
 services:
-  artemis-prometheus:
-    image: prom/prometheus:v2.31.1
-    volumes:
-      - ./prometheus/:/etc/prometheus/
-    command:
-      - '--config.file=/etc/prometheus/prometheus.yml'
+  prometheus:
+    container_name: artemis-prometheus
+    image: prom/prometheus:v2.34.0
+    pull_policy: always
+    # If you want to run this in production, you should persist the /etc/prometheus-directory
+    #volumes:
+      # - ./monitoring/prometheus/:/etc/prometheus/
```

```suggestion
    volumes:
      - ./monitoring/prometheus/:/etc/prometheus/
```
I guess those lines are always needed since they define the configuration which tell Prometheus where to fetch stats from.

---

@sleiss was there any special reason why this is commented out for non-production use cases?

---

Talked to Simon, and he doesn't know why he left this commented, but said it should be uncommented. I just tested it as proposed. Also tested if it works with an Artemis container, but guess the monitoringIp field then has problems if we don't want to use fixed IP addresses. Didn't investigate a solution for this, as it's not urgent. Also added if setups are just working out of the box for the containerized or non-containerized version of Artemis.